src/extensions/dao/dao.extension.service.ts

```diff
@@ -497,7 +497,7 @@ export class DAOService extends BaseService {
           polls.forEach(poll => {
             response += `Link: https://discord.com/channels/${poll.discord_guild_id}/${poll.discord_channel_id}/${poll.discord_message_id}\n`
             response += `ID: ${poll.discord_message_id}\n`
-            response += `Until: ${poll.until}\n`
+            response += `Until: ${poll.until} UTC\n`
```

Comment:
Are we sure this is always using UTC time? I am doubting of this but it could! If we are it's ok.